### PR TITLE
[ALMATY-2022] Add Almaty event to the website

### PR DESCRIPTION
**Pull request by tabu-a**

### CODE REVIEW

**Review by phrawzty — CHANGES_REQUESTED** [review]
p.s. I love the logo! Please send me a t-shirt. ;)

File: data/events/2022-almaty.yml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    linkedin: "https://www.linkedin.com/in/ruspanov/"
+  - name: "Nurbek Sadykov"
+    linkedin: "https://www.linkedin.com/in/nurbeksadykov/"
+
+organizer_email: "almaty@devopsdays.org" # Put your organizer email address here
+
+# List all of your sponsors here along with what level of sponsorship they have.
+# Check data/sponsors/ to use sponsors already added by others.
+sponsors:
+  - id: PS
```
[the remark this answers is not in the document]
Comment (tabu-a) [reply data/events/2022-almaty.yml]: fixed - thanks!

File: data/events/2022-almaty.yml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+#   variable: 2019-01-04T00:00:00+02:00
+#   variable: 2019-01-05T23:59:59+02:00
+# Note: we allow 2022-MM-DD for backward compatibility, but it can lead to unexpected behaviors (like your event disappearing from the front page during your last day)
+
+startdate:  2022-11-11T09:00:00+06:00 # The start date of your event. Leave blank if you don't have a venue reserved yet.
+enddate:  2022-11-11T18:00:00+06:00 # The end date of your event. Leave blank if you don't have a venue reserved yet.
+
+# Leave CFP dates blank if you don't know yet, or set all three at once.
+cfp_date_start:  2022-09-01T09:00:00+06:00 # start accepting talk proposals.
+cfp_date_end:  2022-10-15T09:00:00+06:00 # close your call for proposals.
```
[the remark this answers is not in the document]
Comment (tabu-a) [reply data/events/2022-almaty.yml:18]: I've added a link to a form for speakers on our landing page. Thanks!

**Comment by tabu-a** [comment]
hey @phrawzty - thanks for comments. could you please dm me your shipping address and your t-shirt size on slack? 🙂

**Review by phrawzty — APPROVED** [review]
The Netlify preview isn't regenerating but I think this PR is safe. :)

**Pull request merged**